src/services/utils.ts

```diff
@@ -181,4 +181,9 @@ export class Utils {
 
     documentElement?.style.setProperty(varName, value);
   }
+
+  public static uniqById<T>(collection: Array<T & {id: string}>): Array<T & {id: string}> {
```

Comment:
I would add a type condition to this generic so that only objects that have an actual `id` property can be passed:
```
public static uniqById<T extends {id: string}>(collection: Array<T>): Array<T> {
```